### PR TITLE
Fix to restart Nginx in FCGId mode

### PR DESCRIPTION
**Pull request by iliajie**
Fix to restart Nginx in FCGId mode

### CODE REVIEW

File: phpini/phpini-lib.pl [hunk 1]
```diff
@@ -308,6 +308,15 @@ sub graceful_apache_restart
 		&virtual_server::pop_all_print();
 		}
 	}
+if ($file && &get_config_fmt($file) eq "ini" &&
+	&foreign_installed("virtual-server") && 
+	&foreign_installed("virtualmin-nginx")) {
+	&foreign_require("virtual-server");
+	&foreign_require("virtualmin-nginx", "virtual_feature.pl");
+	my @dom = grep { $file =~ /^$_->{'home'}\// }
```
[the remark this answers is not in the document]
Comment (iliajie) [reply phpini/phpini-lib.pl]: Right, thanks! Fixed.

File: phpini/phpini-lib.pl [hunk 2]
```diff
@@ -308,6 +308,15 @@ sub graceful_apache_restart
...
 		}
 	}
+if ($file && &get_config_fmt($file) eq "ini" &&
+	&foreign_installed("virtual-server") && 
+	&foreign_installed("virtualmin-nginx")) {
+	&foreign_require("virtual-server");
+	&foreign_require("virtualmin-nginx", "virtual_feature.pl");
+	my @dom = grep { &is_under_directory($_->{'home'}, $file) } 
+	              &virtual_server::list_domains();
+	&virtualmin_nginx::feature_restart_web_php($dom[0]);
```
Comment (jcameron) [inline phpini/phpini-lib.pl:318]: What if @doms is empty?

Comment (iliajie) [reply phpini/phpini-lib.pl]: > What if @doms is empty?

It really shouldn't be the case, but okay, sure i checked in a fix for that.

Comment (jcameron) [reply phpini/phpini-lib.pl]: Thanks!

**Comment by iliajie** [comment]
I was making a final check on this PR, and got pretty confused with its name.. as we're not restarting Nginx but FCGId process..

Anyway. All seems good to me to merge it.

**Pull request merged**